src/ContentTypeListener.php

```diff
@@ -153,8 +153,7 @@ public function onFinish(MvcEvent $e)
     public function decodeJson($json)
     {
         // Trim whitespace from front and end of string to avoid parse errors
-        $json = preg_replace('/^\s*/m', '', $json);
-        $json = preg_replace('/\s*/m', '', $json);
+        $json = trim($json);
```

Comment:
`trim()` isn't aggressive, as it doesn't do more than one character on either side, which is why I had `preg_replace()` here. The problem, which #39 correctly identifies, is that I forgot the end-of-string anchor (`$`) on the second call to the function.